components/transactions/graphql/fragments.js

```diff
@@ -72,6 +72,16 @@ export const transactionsQueryCollectionFragment = gqlV2/* GraphQL */ `
         type
         imageUrl
         isIncognito
+        ... on Event {
+          parentCollective {
```

Comment:
`parentCollective` is deprecated

```suggestion
          parent {
```